website/src/images/logo.svg

```diff
@@ -1 +1 @@
-<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 34.3 39"><style>.st0{fill:#F27923;} .st1{fill:#F5A730;} .st2{fill:#20AD92;} .st3{fill:#D0D1D1;}</style><g id="XMLID_1_"><path id="XMLID_75_" class="st0" d="M1.4 29.2c-.2.1-.4.4-.4.7 0 .3.2.6.4.7l14.9 7c.2.1.4.1.6.1V22.1c-.2 0-.4 0-.7.1l-14.8 7z"/><path id="XMLID_74_" class="st1" d="M17 22.1v15.6c.2 0 .5 0 .7-.1l14.9-7c.3-.1.4-.4.4-.7 0-.3-.2-.6-.4-.7l-14.9-6.9c-.3-.1-.5-.2-.7-.2z"/><path id="XMLID_73_" class="st2" d="M1 8.8c0 .3.2.6.4.7l14.9 7c.2.1.4.1.6.1.2 0 .4 0 .6-.1.1-.1.1-.1.4-.2 1.5-.7.5-.2 14.6-6.8.3-.1.5-.4.5-.7 0-.3-.2-.6-.4-.7l-15-7c-.4-.2-.9-.2-1.3 0l-14.9 7c-.2.1-.4.4-.4.7z"/><path id="XMLID_72_" class="st3" d="M17 17.5v4.6c-.2 0-.4 0-.7.1l-14.9 7c-.2.1-.4.4-.4.7V9.3v-.5c0 .3.2.6.4.7l14.9 7c.2.1.4.1.6.1.2 0 .4 0 .6-.1-.4.2-.5.4-.5 1z"/></g></svg>
\ No newline at end of file
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 34.3 39"><style>.st0{fill:rgb(212,4,36);} .st1{fill:rgb(242, 182, 191);} .st2{fill:rgb(16, 28, 40);} .st3{fill:rgb(159, 201, 249);}</style><g id="XMLID_1_"><path id="XMLID_75_" class="st0" d="M1.4 29.2c-.2.1-.4.4-.4.7 0 .3.2.6.4.7l14.9 7c.2.1.4.1.6.1V22.1c-.2 0-.4 0-.7.1l-14.8 7z"/><path id="XMLID_74_" class="st1" d="M17 22.1v15.6c.2 0 .5 0 .7-.1l14.9-7c.3-.1.4-.4.4-.7 0-.3-.2-.6-.4-.7l-14.9-6.9c-.3-.1-.5-.2-.7-.2z"/><path id="XMLID_73_" class="st2" d="M1 8.8c0 .3.2.6.4.7l14.9 7c.2.1.4.1.6.1.2 0 .4 0 .6-.1.1-.1.1-.1.4-.2 1.5-.7.5-.2 14.6-6.8.3-.1.5-.4.5-.7 0-.3-.2-.6-.4-.7l-15-7c-.4-.2-.9-.2-1.3 0l-14.9 7c-.2.1-.4.4-.4.7z"/><path id="XMLID_72_" class="st3" d="M17 17.5v4.6c-.2 0-.4 0-.7.1l-14.9 7c-.2.1-.4.4-.4.7V9.3v-.5c0 .3.2.6.4.7l14.9 7c.2.1.4.1.6.1.2 0 .4 0 .6-.1-.4.2-.5.4-.5 1z"/></g></svg>
```

Comment:
I just changed the `fill` colors. It now looks like this

<img width="684" alt="image" src="https://user-images.githubusercontent.com/1110551/56850094-cf925880-68fd-11e9-94dd-e46be4a1d504.png">